ReactWindows/ReactNative.Shared/Views/TextInput/ReactTextBox.cs

```diff
@@ -104,6 +134,46 @@ public int IncrementEventCount()
             return Interlocked.Increment(ref _eventCount);
         }
 
+        protected override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+            if (DeleteButton == null)
+            {
+                DeleteButton = (Button)GetTemplateChild("DeleteButton");
+                if (DeleteButtonVisibilityToken != null)
+                {
+                    DeleteButton.UnregisterPropertyChangedCallback(Button.VisibilityProperty, DeleteButtonVisibilityToken ?? 0);
```

Comment:
>DeleteButtonVisibilityToken ?? 0 [](start = 94, length = 32)

maybe (int)DeleteButtonVisibilityToken is better since it will not swallow a possible bug of null getting here?

Reply:
Do you mean (long)DeleteButtonVisibilityToken?

Reply:
Yes, sorry for mistake.